examples/turn-server/lt-cred-turn-rest/main.go

```diff
@@ -0,0 +1,72 @@
+// Package main implements a TURN server using
```

Comment:
Please add copyright stanza as per the failing REUSE test. See `examples/turn-server/lt-cred/main.go` for an example.